cfgov/unprocessed/css/pages/beta.less

```diff
@@ -13,10 +13,14 @@
     font-size: 0.875em;
     transition: padding 0.2s ease-out;
 
-    &.expandable__expanded {
+    &.m-expandable__expanded {
         padding: @grid_gutter-width 0;
     }
 
+    .m-expandable_content {
+        border-bottom: none;
```

Comment:
So if I don't remove the border then it looks like:

<img width="1237" alt="screen shot 2016-01-27 at 12 53 05 pm" src="https://cloud.githubusercontent.com/assets/1696212/12622794/0625d508-c4f5-11e5-99ea-abd11864cb5d.png">